packages/server/src/index.ts

```diff
@@ -13,15 +13,22 @@ import { authPlugin } from './plugins/auth-plugin.js';
 async function main() {
   const application = await createGraphQLApp(env);
 
+  // While createGraphQLApp initializes the application, we use buildSubgraphSchema to build the Subgraph Schema.
   const yoga = createYoga({
+    schema: application.schema,
+    graphqlEndpoint: '/legacy',
+    graphiql: {
+      title: 'Legacy Subgraph',
+    },
+    landingPage: true,
     plugins: [
       authPlugin(),
       adminContextPlugin(),
       useGraphQLModules(application),
       useDeferStream(),
       useHive({
```

Comment:
The subgraph is now considered "dumb". Meaning it should not deal with Hive directly. Please move this to the gateway setup, so the Gateway is in charge or reporting operations to Hive.